### ISSUE TITLE
🔬 Manual accessibility test/review - Record of decision pattern

### ISSUE THREAD

**Issue by RM-1978** [GCTC-NTGC/gc-digital-talent#8834]
### 🕵️ Details

Manual evaluation of a selection of pages and components for A11y in the_**_Record of Decision_** pattern on UAT.  

## 🔅 WCAG 2 Criteria

- [Diverse Abilities and Barriers ](https://www.w3.org/WAI/people-use-web/abilities-barriers/#cognitive)
- [Understanding the Current WCAG 2 standard ](https://www.w3.org/WAI/WCAG22/Understanding/)

## 🧪 Testing 

**Testing configurations**

- Operating Systems: Windows 11, macOS, iOS, Android 
- Browser: Edge, Safari, Chrome
- Assistive Technology: Narrator, VoiceOver, Talkback

### **Testing steps**

1. Turn on AT.
2. Open the ROD user interface.
3. Test for proper landmarks roles and labels (e.g., Main, Nav).
4. Confirm that all components/widgets have appropriate ARIA roles.
5. Review page titles and headings for descriptive titles and proper structure.
6. Review reading order and focus management (focus order, component triggers, modals, etc.). 
7. Review non-text elements for information or functional value and proper placement.
8. Confirm that non-text elements with informative or functional value have meaningful alternative text or ARIA labels.
9. Review colour contrast for a contrast ratio of 4:5:1 between foreground and background. 
10. Confirm link texts are descriptive and that their "underline" signifier is present. Alternatively, they should have a contrast ratio of 3:1.
11. Review colour contrast for perceived contrast in addition to computed values. 
12. Confirm that meaningful graphical elements (those with informational value) have a contrast value of 3:1 
13. Confirm that key pages do not rely solely on colour to convey meaning or instruction. 
14. Confirm that touch targets are usable.
17. Check French-language pages for proper default language.  
18. Confirm that the ROD interface does not require users to re-enter information they have already provided (e.g., in a previous step of the RoD process).
19. Plain language check: 
    - [ ] Check that the copy is presented concisely.
    - [ ] Check that documents are broke into separate topics using semantic headers.
    - [ ] Use short lists and bullets to organize information.
    - [ ] Use even more headings with less under each heading.
    - [ ] Flag if information on each page is presented in more than two semantic heading levels.
    - [ ] Check that page titles and copy are crafted with key words that our readers would use when doing a web search for our product.
    - [ ] Verify that readers who do not have knowledge of our platform can read related pages on the site. 
    - [ ] Check that copy explains things succinctly so each page can stand on its own.
    - [ ] Check that link language should describe what your reader will get if they click the link (e.g., Never use “click here” or “ Learn more” as links).
    - [ ] Verify that unnecessary or redundant words have been eliminated.

## 🦋 Expected Behaviour

- AT users can build an assessment plan with the same ease as visual/auditory users. 
- AT users can conduct and manage candidate assessments with the same as visual/auditory users.  

## ✅ Acceptance Criteria
A set of assumptions which, when tested, verify that the issue was addressed.
- [ ] Users can complete RoD tasks using ATs. 
- [ ] List of GitHub issues is created to address any bugs or issues.

**Comment by petertgiles** [comment]
Hi Lee.  Unless you'd like me to get involved I'll let you manage this one.

**Comment by RM-1978** [comment]
@petertgiles I am wondering if we should co-manage this one. As scrum master, please let me know when the ROD is ready for review and hold me to the timelines.  We always want you involved in testing, and  of course and it would be great to have your help  to stay on track and make sure it makes it into scrum.

**Comment by tristan-orourke** [comment]
Testing seems to be done, resulting tickets coming soon

**Comment by RM-1978** [comment]
Sorry @tristan-orourke, my understanding is that we only did testing with an AT user.  I think we should check in with @esizer because i don't remember doing my part of the manual accessibility review yet.

**Comment by RM-1978** [comment]
Notes sent on to  @esizer and @Jerryescandon

**Comment by RM-1978** [comment]
https://docs.google.com/document/d/1vJEnSfsAw0bBz-agcMnfz8u3Z95zvm0w/edit?usp=drive_link&ouid=108801801999372031379&rtpof=true&sd=true
